client/src/classes/AudioVisualizer.ts

```diff
@@ -33,13 +37,16 @@ export class AudioVisualizer {
     }
   }
 }
+
+export const isSignificantlyDifferent = (
+  prev: number,
+  curr: number,
+  threshold: number
+): boolean => {
+  return Math.abs(prev - curr) > threshold;
+};
+
 export const gainToMultiplier = (gain: number): number => {
-  const max = 150;
-  let _multiplier = 0;
-  if (gain < max) {
-    _multiplier = gain / max;
-  } else {
-    _multiplier = 1;
-  }
-  return _multiplier;
+  const max = 250;
+  return gain < max ? gain / max : 1;
```

Comment:
It's more intuitive using Math.min(gain/max, 1). Just a personal preference.

Reply:
I agree